syz-ci/syz-ci.go

```diff
@@ -111,6 +111,8 @@ type Config struct {
 	Managers     []*ManagerConfig `json:"managers"`
 	// Poll period for jobs in seconds (optional, defaults to 10 seconds)
 	JobPollPeriod int `json:"job_poll_period"`
+	// Set up a second (parallel) job processor to speed up processing.
```

Comment:
Say that it's doing only parallel patch testing.
And perhaps call it `ParallelPatchTesting`.
Also if you want incremental roll out, shouldn't this be per manager?